parser/parser.y

```diff
@@ -1998,60 +1998,67 @@ DoStmt:
  *
  *******************************************************************/
 DeleteFromStmt:
-	"DELETE" PriorityOpt QuickOptional IgnoreOptional "FROM" TableName IndexHintListOpt WhereClauseOptional OrderByOptional LimitClause
+	"DELETE" TableOptimizerHints PriorityOpt QuickOptional IgnoreOptional "FROM" TableName IndexHintListOpt WhereClauseOptional OrderByOptional LimitClause
```

Comment:
Seem that the `TableOptimizerHints` is not used?

Reply:
Yeah, but it will cause yacc reduce conflict without this `TableOptimizerHints`.